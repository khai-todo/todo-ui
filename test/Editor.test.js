import Subject from '../src/Editor'
import snap from './lib/snapshot'

it('empty rendering', () => snap(<Subject />))
