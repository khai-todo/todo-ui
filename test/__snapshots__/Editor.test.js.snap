// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`empty rendering 1`] = `<div />`;
